notebooks/case2_observations_analysis.md

```diff
@@ -465,22 +480,41 @@ n_plot_sex.head()
 ```{code-cell} ipython3
 :clear_cell: true
 
-pivoted = n_plot_sex.pivot_table(columns="sex", index="verbatimLocality", values="count")
+pivoted = n_plot_sex.pivot(columns="sex", index="verbatimLocality", values="count")
 pivoted.head()
 ```
 
 As such, we can use the variable `pivoted` to plot the result:
 
 ```{code-cell} ipython3
-:clear_cell: false
-
 pivoted.plot(kind='bar', figsize=(12, 6), rot=0)
 ```
 
 <div class="alert alert-success">
 
 **EXERCISE**
 
+Recreate the previous plot with the `catplot` function from the Seaborn library starting from `n_plot_sex`.
+
+<details><summary>Hints</summary>
+
+- Check the `kind` argument of the `catplot` function to figure out to specify you want a barplot with given x and y values.
+- To link a column to different colors, use the `hue` argument
+
+
+</details>
+
+```{code-cell} ipython3
+:clear_cell: true
+
+sns.catplot(data=n_plot_sex, x="verbatimLocality", y="count",
+            hue="sex", kind="bar", height=3, aspect=3)
```

Comment:
Added a variation on "recreate with seaborn", but starting with the intermediate (calculating counts ourselves, but without the reshaping that pandas plot() needs)